Dockerfile

```
@@ -2,14 +2,17 @@ FROM python:3.7
 
 WORKDIR /opt/app
 
+ARG VERSION_STRING
```

Some [docker image repos](https://github.com/github/docs/issues/2617) can surface docker image/container metadata. We'll probably want this docker image label, to add some metadata about the image. Sorry I haven't been consistently using it!
```suggestion
LABEL org.opencontainers.image.source https://github.com/uwcirg/helloworld-confidential-backend-sof
ARG VERSION_STRING
```